### PR TITLE
update ET email client local test to send all emails for all products

### PR DESCRIPTION
**Pull request by johnduffell**
reliased I didn't push these changes.

Basically before the unique key on the emails was all payment id.  This was changed to be invoice id for the overdue email, because there's no actual payment for an overdue invoice.  There may be several.
The code was duplicated which was alright, but then I wanted to test over all the possible products, as ET will send different emails.  To avoid the duplication getting worse I just combined into one function and called that number of emails * number of products times, with the right parameters.

This is just a test you'd run locally, but it makes sense to have the latest version pushed.  It won't affect anythign that happens automatically.

@jacobwinch

### CODE REVIEW

File: src/test/scala/manualTest/EmailClientSystemTest.scala [hunk 1]
```diff
@@ -66,15 +55,24 @@ object EmailClientSystemTest extends App {
...
+  } yield Seq(
+    "Supporter",
+    "Digital Pack",
+    "Guardian Weekly Zone A",
+    "Guardian Weekly Zone B",
+    "Guardian Weekly Zone C",
+    "Contributor",
+    "Newspaper Voucher",
+    "Newspaper Delivery"
+  ).flatMap(xxx => five(a, xxx)).foreach {
```
Comment (jacobwinch) [inline src/test/scala/manualTest/EmailClientSystemTest.scala]: xxx?

File: src/test/scala/manualTest/EmailClientSystemTest.scala [hunk 2]
```diff
@@ -66,15 +55,24 @@ object EmailClientSystemTest extends App {
     config <- Config.parseConfig(configAttempt)
     deps = EmailSendStepsDeps.default(Stage("CODE"), RawEffects.createDefault.response, config.etConfig)
     a = config.etConfig.etSendIDs
```
Comment (jacobwinch) [inline src/test/scala/manualTest/EmailClientSystemTest.scala]: What is 'a'?

File: src/test/scala/manualTest/EmailClientSystemTest.scala [hunk 3]
```diff
@@ -37,27 +37,16 @@ object EmailClientSystemTest extends App {
...
-          card_expiry_date = "cardExpiryValue",
-          first_name = s"firstNameValue overdue",
-          last_name = "lastNameValue",
-          primaryKey = InvoiceId(s"invoiceId$unique"), // must be unique otherwise the email won't arrive
-          price = "49.0 GBP",
-          serviceStartDate = "31 January 2016",
-          serviceEndDate = "31 January 2017"
-        )
-      )
+  def five(a: ETSendIds, product: String) = {
```
Comment (jacobwinch) [inline src/test/scala/manualTest/EmailClientSystemTest.scala]: I think this function could have a clearer name

**Comment by jacobwinch** [comment]
Looks good - just a couple of comments about naming (as per Scala Exchange advice!)

**Comment by johnduffell** [comment]
haha 🙈

**Comment by jacobwinch** [comment]
@johnduffell - shall we merge this (👍  from me)?

**Comment by johnduffell** [comment]
need to do the review comments first, sorry

**Comment by johnduffell** [comment]
finally merged this one 😆 🚀

**Pull request merged**